packages/appstore/src/components/onboarding/static-cfd-account-manager.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { Button, Text } from '@deriv/components';
+import { formatMoney, CFD_PLATFORMS } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import WalletIcon from 'Assets/svgs/wallet';
+import { TPlatform } from 'Types';
+
+import './static-cfd-account-manager.scss';
+import classNames from 'classnames';
+
+type TStaticCFDAccountManager = {
+    type: string;
+    appname: string;
+    loginid?: string;
+    currency?: string;
+    description?: string;
+    is_animated?: boolean;
+    has_account?: boolean;
+    is_last_step?: boolean;
+    derived_amount?: string;
+    is_icon_blurry?: boolean;
+    is_item_blurry?: boolean;
+    is_text_blurry?: boolean;
+    is_get_blurry?: boolean;
+    is_topup_blurry?: boolean;
+    is_trade_blurry?: boolean;
```

Comment:
```suggestion
    is_icon_blurry?: boolean;
    is_item_blurry?: boolean;
    is_text_blurry?: boolean;
    is_get_blurry?: boolean;
    is_topup_blurry?: boolean;
    is_trade_blurry?: boolean;
```
We can consolidate this to an object in this structure:
```
is_blurry: {
    icon: boolean;
    item: boolean;
    text: boolean;
    get: boolean;
    topup: boolean;
    trade: boolean;
}
```